crates/coingecko/src/client.rs

```diff
@@ -71,13 +71,13 @@ impl CoinGeckoClient {
     }
 
     pub async fn get_global(&self) -> Result<Global, Box<dyn Error + Send + Sync>> {
-        let result = self.client.get("/api/v3/global").await;
+        let result = self.client.get("/api/v3/global", None::<&()>).await;
```

Comment:
```suggestion
        let result = self.client.get("/api/v3/global").await;
```

I should be able to fetch with no params, clean